proto/message_contents/invitation.proto

```diff
@@ -19,13 +19,19 @@ message InvitationV1 {
         bytes key_material = 1;  // randomly generated key material (32 bytes)
     }
 
+    message GroupContext {
```

Comment:
Is there a reason we can't just do this with the existing `metadata` field? While, the value is required to be a string, it doesn't seem like the most onerous thing in the world to just have it be a comma separated list

Reply:
I considered metadata for this, but it felt a bit too loose to me for such a core concept as group membership. I could be convinced otherwise but to me it seemed like a good idea to have something concrete in the protocol to determine if something is a group or not.